lib/util/include/hse/util/bloom_filter.h

```diff
@@ -21,6 +20,7 @@
  */
 #define BF_BKTSHIFT (9)
 #define BF_ROTL (11)
+#define BF_BYTE_SHIFT (3)
```

Comment:
You could move `BF_BKTSHIFT` and `BK_ROTL` into `bloom_filter.c`.

Reply:
Sure thing. Thanks for pointing it out.